adserver/views.py

```diff
@@ -1177,10 +1177,11 @@ def get_context_data(self, **kwargs):  # pylint: disable=too-many-locals
                 publishers_and_reports.append((publisher, report))
 
         # Sort reports by revenue
-        if sort == "revenue":
+        sort_options = report["total"].keys()
```

Comment:
By passing this directly to the display, it does result in slightly awkward display but it is just for us so I guess it's ok. I'm not sure every field is useful either.

![Screen Shot 2020-11-03 at 11 35 39 AM](https://user-images.githubusercontent.com/185043/98032388-cb83a880-1dc8-11eb-9c3b-d6131208c18a.png)

Reply:
Yea, it's ugly for was easy enough. I'd also like more complex filtering (eg. views__gt=500) or similar, but baby steps :)